src/catacharset.cpp

```diff
@@ -1,16 +1,17 @@
 #include "catacharset.h"
 
+#include <algorithm>
+#include <array>
 #include <cstdlib>
 #include <cstring>
-#include <array>
 
 #include "options.h"
 #include "output.h"
 #include "wcwidth.h"
 
 #if defined(_WIN32)
```

Comment:
```suggestion
#if defined(_WIN32)
#include "platform_win.h"
```
```

Reply:
Should not be reordered alphabetically and `platform_win.h` should be included before `mmsystem.h`.